File: Python/Program/Speaker.py
from Periferal import Periferal

class Speaker(Periferal):
    def __init__(self, merk, nama, koneksi, dayaWatt):
        super().__init__(merk, nama, koneksi)
        self.dayaWatt = dayaWatt
        
    def setDayaWatt(self, dayaWatt):
        self.dayaWatt = dayaWatt
        
    def getDayaWatt(self):
        return self.dayaWatt
    
    

File: Java/Program/Speaker.java
public class Speaker extends Periferal {
    private int dayaWatt;

    public Speaker(String merk, String nama, String koneksi, int dayaWatt) {
        super(merk, nama, koneksi);
        this.dayaWatt = dayaWatt;
    }

    public void setDayaWatt(int dayaWatt) {
        this.dayaWatt = dayaWatt;
    }

    public int getDayaWatt() {
        return dayaWatt;
    }
}